pkgs/test_api/lib/src/remote_listener.dart

```diff
@@ -135,7 +134,7 @@ class RemoteListener {
               // useful errors when calling `test()` and `group()` within a test,
               // and so they can add to the declarer's `tearDownAll()` list.
               zoneValues: {#test.declarer: declarer});
-        }, onError: (error, StackTrace stackTrace) {
+        }, (Object error, StackTrace stackTrace) {
```

Comment:
[nit] we can remove the argument types here now - one of the benefits of `runZonedGuarded` is we have an _actual_ function type with arguments.
```suggestion
        }, (error, stackTrace) {
```